src/main/java/core/service/ReportServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.service;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String CHAR_FOR_SPLIT = ",";
+
+    @Override
+    public String create(Map<String, Integer> data) {
+        return data.entrySet()
+            .stream()
+            .map(stringIntegerEntry -> stringIntegerEntry.getKey() + CHAR_FOR_SPLIT
+                    + stringIntegerEntry.getValue())
+            .collect(Collectors.joining(System.getProperty("line.separator")));
```

Comment:
```suggestion
            .collect(Collectors.joining(System.lineSeparator()));
```
will this work the same?